test/integration/virtual-rules/index.js

```diff
@@ -0,0 +1,315 @@
+describe('virtual rules', function() {
+	describe('checks', function() {
```

Comment:
I don't think we should split this up into checks. For integrations, the checks don't matter. Just have rules with different passes and fails.